tsdb/chunks/head_chunks_test.go

```diff
@@ -188,51 +185,41 @@ func TestHeadReadWriter_Truncate(t *testing.T) {
 		return mint
 	}
 
-	cutFile := func(i int) {
+	cutFile := func(i int) int64 {
 		testutil.Ok(t, hrw.CutNewFile())
-
-		mint := addChunk()
-
-		if i == startIndexAfter1stTruncation {
-			timeToTruncate = int64(mint)
-		} else if i == startIndexAfter2ndTruncation {
-			timeToTruncateAfterRestart = int64(mint)
-		}
-	}
-
-	// Cut segments.
-	for i := 1; i <= totalFiles; i++ {
-		cutFile(i)
+		return int64(addChunk())
 	}
 
 	// Verifying the files.
```

Comment:
This comment is redundant with the name of the function in the next line. I'd just leave it out.